esp/templates/errors/program/notateacher.html

```diff
@@ -17,7 +17,8 @@ <h1>Not a Teacher!</h1>
 <br />
 Thank you for your understanding.
 <br /><br />
-<em>If you feel you are a valid teacher, and you still are getting this error message, please email us at <u>{{ DEFAULT_EMAIL_ADDRESSES.support }}</u></em>.
+<em>If you think you should be registered as a valid {{ settings.INSTITUTION_NAME }} {{ settings.ORGANIZATION_SHORT_NAME }}teacher,
+and you still are getting this error message, please email us at <u>{{ DEFAULT_EMAIL_ADDRESSES.support }}</u></em>.
 <br />
-<a href="/myesp/signout/?redirect=/myesp/register/&role=Teacher" title="Create a teacher account!">Click here to create a new account for an ESP Teacher!</a>
+<a href="/myesp/signout/?redirect=/myesp/register/&role=Teacher" title="Create a teacher account!">Click here to create a new account {{ settings.INSTITUTION_NAME }} {{ settings.ORGANIZATION_SHORT_NAME }} teacher account!</a>
```

Comment:
account twice

Reply:
This should also probably be the same as the one for students (above)

Reply:
thanks, I got it right on the student one

Reply:
You have INSTITUTION_NAME here but not for student, that's what I meant by my second comment. Would be good to be consistent.

Reply:
ahh gotcha